scripts/libs.versions.toml

```diff
@@ -3,28 +3,25 @@
 # Do not add a dependency to an individual sample, edit the global version instead.
 #####
 [versions]
-accompanist = "0.34.0"
-androidGradlePlugin = "8.5.0"
-androidx-activity-compose = "1.9.0"
+accompanist = "0.36.0"
+androidGradlePlugin = "8.6.0"
+androidx-activity-compose = "1.9.2"
 androidx-appcompat = "1.7.0"
 androidx-benchmark = "1.2.4"
 androidx-benchmark-junit4 = "1.2.4"
-androidx-compose-bom = "2024.06.00"
-androidx-compose-latest = "1.7.0-beta07"
-androidx-compose-material3 = "1.3.0-beta05"
-androidx-compose-material3-adaptive = "1.0.0-beta04"
+androidx-compose-bom = "2024.09.00"
 androidx-constraintlayout = "1.1.0-alpha13"
 androidx-core-splashscreen = "1.0.1"
 androidx-corektx = "1.13.1"
 androidx-glance = "1.1.0"
 androidx-lifecycle = "2.8.2"
-androidx-lifecycle-compose = "2.6.1"
-androidx-lifecycle-runtime-compose = "2.6.1"
-androidx-navigation = "2.7.7"
+androidx-lifecycle-compose = "2.8.5"
+androidx-lifecycle-runtime-compose = "2.8.5"
+androidx-navigation = "2.8.0"
 androidx-palette = "1.0.0"
-androidx-test = "1.5.0"
-androidx-test-espresso = "3.5.1"
-androidx-test-ext-junit = "1.1.5"
+androidx-test = "1.6.1"
+androidx-test-espresso = "3.6.1"
+androidx-test-ext-junit = "1.2.1"
 androidx-test-ext-truth = "1.5.0"
 androidx-tv-foundation = "1.0.0-alpha10"
 androidx-tv-material = "1.0.0-beta01"
```

Comment:
I believe both of these tv libraries can be updated to the 1.0.0 stable that came out a couple of weeks ago.

Reply:
Seems that tv-material is stable but tv-foundation is still alpha. I've updated